concepticondata/conceptlists/Luangthongkum-2019-341.tsv

```diff
@@ -0,0 +1,342 @@
+ID	NUMBER	ENGLISH	CONCEPTICON_ID	CONCEPTICON_GLOSS
+Luangthongkum-2019-341-1	1	all, all gone, whole		
+Luangthongkum-2019-341-2	2	ant	587	ANT
+Luangthongkum-2019-341-3	3	arrive	1387	ARRIVE
+Luangthongkum-2019-341-4	4	arrow, dart	977	ARROW
+Luangthongkum-2019-341-5	5	ascend, go up, rise	3207	GO UP OR RISE (ONESELF)
+Luangthongkum-2019-341-6	6	ashes, fireplace		
+Luangthongkum-2019-341-7	7	astringent		
```

Comment:
That's a good candidate for a new concept, do we currently have a place where we collect concept ideas?

Reply:
Yes, here: https://github.com/concepticon/pyconcepticon/issues/20